src/libraries/Microsoft.PowerFx.Connectors/Shared/ServiceFunction.cs

```diff
@@ -287,17 +287,17 @@ public override bool CheckForDynamicReturnType(TexlBinding binding, TexlNode[] a
         }
 #endif
 
-        public override async Task<List<string>> GetConnectorSuggestionsAsync(CallNode callNode, int argPosition, CancellationToken cts)
+        public override async Task<List<RecordValue>> GetConnectorSuggestionsAsync(FormulaValue[] knownParameters, int argPosition, CancellationToken cts)
```

Comment:
>RecordValue [](http://example.com/codeflow?start=40&length=11)

RecordValue doesn't seem like right return type. 
Ie, these are not values that will flow into evaluator.